### PR TITLE
The Ember Times No. 190 - October 8th 2021 

### PR DESCRIPTION
**Pull request by amyrlam**
👋 New to The Ember Times? If you want to claim a topic to write about, please comment here on the PR or chat with us in [#support-ember-times](https://discordapp.com/channels/480462759797063690/485450546887786506) on the [Ember Discord](https://discord.gg/emberjs)! 

***

### Addon/App releases

- [x] FYI - ember-cli@4.0.0-beta.1 was released yesterday (includes a few big features: pnpm support for ember new; defaults to GitHub Actions for CI; updates to ember-auto-import@2; ???) (:lock: @ijlee2)
  - https://github.com/ember-cli/ember-cli/releases/tag/v4.0.0-beta.1
  - https://discord.com/channels/480462759797063690/485450546887786506/894963110090272798
- [x] Hacktoberfest (:lock: @jenweber I can help, just not sure where we're at rn)
  - here are 81 repositories participating, it looks like: https://github.com/search?q=topic%3Aember+topic%3Ahacktoberfest
  - I win recently updated! https://github.com/search?o=desc&q=topic%3Aember+topic%3Ahacktoberfest&s=updated&type=Repositories

## 📆  Future stories

- [ ] EmberFest once videos live

## 🤔  Questions?

See #support-ember-times on Discord or CONTRIBUTING.

## ✅  Publishing TODOs:
- [ ] Check Netlify / localhost preview, to ensure display e.g. buttons on emberjs.com/blog is good
- [ ] Crosspost on dev.to: delete any alex ignore's, manually redo Readers Question picture, add Canonical URL 
- [ ] Set up tweet for Monday morning on tweetdeck
- [ ] Post $natural for #news-and-announcements in Discord https://github.com/ember-learn/ember-blog/blob/main/CONTRIBUTING.md#publishing-an-issue-of-the-ember-times
- [ ] Shoutout the real MVP's, our writers!

### CODE REVIEW

**Comment by amyrlam** [comment]
- we might push the issue to friday, october 15
- we could ensure that the branch is set up for hacktoberfest contribution count (i think we might have changed the year to 2022 and had folks merge into `main` for that in the past?

**Comment by amyrlam** [comment]
let's keep it for this friday, if we get hacktoberfest in. can help w/hacktoberfest writeup / grooming of issues ... just not sure what the lay of the land is with that yet!

**Comment by jenweber** [comment]
@amyrlam we don't have many curated issues at the moment, but we do know that there is a lot of work to do to help addons get ready for 4.0 and so I think we can leverage that in the writeup! https://github.com/ember-learn/ember-blog/pull/1043

File: content/the-ember-times-issue-190.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+
+<add your name to author list, top and bottom>
+<add short title to "SOME-INTRO-HERE">
+
+---
+
+## [👏 Contributors' corner](https://guides.emberjs.com/release/contributing/repositories/)
+
+<p>This week we'd like to thank our siblings for their contributions to Ember and related repositories! 💖</p>
```
Comment (ijlee2) [inline content/the-ember-times-issue-190.md]:
```suggestion
<p>This week we'd like to thank <a href="https://github.com/ef4" rel="noopener noreferrer" target="_blank">Edward Faulkner (@ef4)</a>, <a href="https://github.com/lfrost" rel="noopener noreferrer" target="_blank">Lyle Frost (@lfrost)</a>, <a href="https://github.com/jherdman" rel="noopener noreferrer" target="_blank">James Herdman (@jherdman)</a>, <a href="https://github.com/jfdnc" rel="noopener noreferrer" target="_blank">Jacob (@jfdnc)</a>, <a href="https://github.com/ddzz" rel="noopener noreferrer" target="_blank">Darius Dzien (@ddzz)</a>, <a href="https://github.com/amyrlam" rel="noopener noreferrer" target="_blank">Amy Lam (@amyrlam)</a>, <a href="https://github.com/SergeAstapov" rel="noopener noreferrer" target="_blank">Sergey Astapov (@SergeAstapov)</a>, <a href="https://github.com/jenweber" rel="noopener noreferrer" target="_blank">Jen Weber (@jenweber)</a>, <a href="https://github.com/chancancode" rel="noopener noreferrer" target="_blank">Godfrey Chan (@chancancode)</a>, <a href="https://github.com/bmish" rel="noopener noreferrer" target="_blank">Bryan Mishkin (@bmish)</a>, <a href="https://github.com/rwjblue" rel="noopener noreferrer" target="_blank">Robert Jackson (@rwjblue)</a>, <a href="https://github.com/chrisrng" rel="noopener noreferrer" target="_blank">Chris Ng (@chrisrng)</a>, <a href="https://github.com/rwwagner90" rel="noopener noreferrer" target="_blank">Robert Wagner (@rwwagner90)</a>, <a href="https://github.com/bertdeblock" rel="noopener noreferrer" target="_blank">Bert De Block (@bertdeblock)</a>, <a href="https://github.com/aoifehannigan" rel="noopener noreferrer" target="_blank">Aoife Hannigan (@aoifehannigan)</a>, <a href="https://github.com/patocallaghan" rel="noopener noreferrer" target="_blank">Pat O'Callaghan (@patocallaghan)</a>, <a href="https://github.com/locks" rel="noopener noreferrer" target="_blank">Ricardo Mendes (@locks)</a>, <a href="https://github.com/fozy81" rel="noopener noreferrer" target="_blank">Tim (@fozy81)</a>, <a href="https://github.com/dcyriller" rel="noopener noreferrer" target="_blank">Cyrille (@dcyriller)</a>, <a href="https://github.com/NullVoxPopuli" rel="noopener noreferrer" target="_blank">@NullVoxPopuli</a>, <a href="https://github.com/nlepage" rel="noopener noreferrer" target="_blank">Nicolas Lepage (@nlepage)</a>, <a href="https://github.com/nlfurniss" rel="noopener noreferrer" target="_blank">Nathaniel Furniss (@nlfurniss)</a>, <a href="https://github.com/Turbo87" rel="noopener noreferrer" target="_blank">Tobias Bieniek (@Turbo87)</a>, <a href="https://github.com/chriskrycho" rel="noopener noreferrer" target="_blank">Chris Krycho (@chriskrycho)</a>, and <a href="https://github.com/ijlee2" rel="noopener noreferrer" target="_blank">Isaac Lee (@ijlee2)</a> for their contributions to Ember and related repositories! 💖</p>
```

**Pull request merged**